app.js

```diff
@@ -139,6 +144,31 @@ const actionHandlerMultipleVerify = (field, singleVerify) => {
   }
 }
 
+const get_price_for_module = () => {
+  return (req, res) => {
+    const module = req.body['module']
+    const cost_code = req.body['cost_code']
+    const product = req.body['product']
+
+    let errors = []
+    if (!verifyModuleName(module)) {
+      errors.push('Module name not valid')
+    }
+    if (!verifyProductName(product)) {
+      errors.push('Product name not valid') 
+    }
+    if (!((verifySubAccountCode(cost_code) || verifyAccountCode(cost_code)))) {
+      errors.push('Project not valid') 
```

Comment:
Could this error be more specific that something is wrong with the (sub-)cost code?